update extension support info (#8054)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/microsoft-defender-application-guard/faq-md-app-guard.md

To help keep the Application Guard Edge session secure and isolated from the host device, we don't copy the Favorites stored in the Application Guard Edge session back to the host device. 

> ### Are Extensions supported in the Application Guard?

Reviewer: ```suggestion
### Are Extensions supported in the Application Guard Edge session?
```

> Extension installs in the container is supported starting from Microsoft Edge version 81. For more details, see [Extension support inside the container](https://docs.microsoft.com/deployedge/microsoft-edge-security-windows-defender-application-guard#extension-support-inside-the-container).

Reviewer: ```suggestion
Extension installs in the container are supported from Microsoft Edge version 81. For more details, see [Extension support inside the container](https://docs.microsoft.com/deployedge/microsoft-edge-security-windows-defender-application-guard#extension-support-inside-the-container).
```
